RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.cc

```diff
@@ -128,7 +135,14 @@ EcalUncalibRecHitWorkerMultiFit::set(const edm::EventSetup& es)
         // for the time correction methods
         es.get<EcalTimeBiasCorrectionsRcd>().get(timeCorrBias_);
 
-        int nnoise = noisecovariances->EBG12SamplesCorrelation.size();
+        int nnoise = SampleVector::RowsAtCompileTime;
+        SampleMatrix &noisecorEBg12 = noisecors_[1][0];
```

Comment:
noisecorEBg12 etc were previously explicitly initialized to SampleMatrix::Zero().
I don't clearly see noisecors_ initialized here (and docs appear to say that the default constructed matrix is uninitialized)